master/getting-started/docker/installation/manual.md

```diff
@@ -8,16 +8,16 @@ Calico runs as a Docker container on each host. The `calicoctl` command line too
 
 1. Download the calicoctl binary:
 
-	 ```
-   sudo wget -O /usr/local/bin/calicoctl http://www.projectcalico.org/builds/calicoctl
```

Comment:
Ah this needs to be reverted. Subcontent for bulleted lists should be indented the same amount as the first letter of the head bullet. In this case, there are 3 characters (`1` `. ` ` `) before the `D`.

If you'd like, those tabs can be converted to spaces so the formatting doesn't look bad in pure markdown.